lib/generators/rolemodel/semaphore/templates/semaphore.yml.erb

```diff
@@ -3,7 +3,7 @@ name: <%= @project_name %>
 agent:
   machine:
     type: e1-standard-2
-    os_image: ubuntu1804
+    os_image: ubuntu2004
```

Comment:
This file still is running a Webpacker compile command. 

Remove this whole block and any dependencies on it:
```yml
  - name: Assets Compile
    dependencies:
      - Build
    execution_time_limit:
      minutes: 20
    task:
      jobs:
        - name: Webpack Compile and Store Assets
          commands:
            - bundle exec rails webpacker:compile
            - cache store assets-public-$SEMAPHORE_WORKFLOW_ID public
```